x-pack/legacy/plugins/code/server/repository_service.ts

```diff
@@ -211,7 +120,11 @@ export class RepositoryService {
         if (files.has(privateKey)) {
           try {
             this.log.debug(`try with key ${privateKey}`);
-            return await action(privateKey);
+            await git.addConfig(
+              'core.sshCommand',
+              `ssh -i ${path.join(this.credsPath, privateKey)}`
```

Comment:
should we use resolve?
I remember from someone says we should try use resolve insttead of join as much as possible